syntax = "proto3";

option java_multiple_files = true;
option java_package = "dna.node.proto";

package dna.node.proto;


message TrackFeatureProto {
    string node_id = 1;
    string track_id = 2;
		repeated float feature = 3;
    oneof optional_zone_relation {
      string zone_relation = 4;
    }
		int64 frame_index = 5;
    int64 ts = 6;
};